modules/local/markdups/meta.yml

```diff
@@ -0,0 +1,62 @@
+name: markdups
+description: Identify and mark duplicate reads ifrom alignment data
+keywords:
+  - duplicating marking
+  - markdups
+tools:
+  - MarkDups:
+      description: Identify and mark duplicate reads ifrom alignment data
+      homepage: https://github.com/hartwigmedical/hmftools/tree/master/mark-dups
+      documentation: https://github.com/hartwigmedical/hmftools/tree/master/mark-dups
```

Comment:
This link doesn't actually work.
Is the tool now called [redux](https://github.com/hartwigmedical/hmftools/tree/master/redux)?

Reply:
Good catch, the MarkDups functionality has been rolled into REDUX but oncoanalyser still currently uses MarkDups - I've updated the URL to a working one in https://github.com/nf-core/oncoanalyser/pull/72.